main.py

```diff
@@ -2,6 +2,38 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres_list = ["Western", "Action", "Dramma"]
+    [Genre.objects.create(name=item, ) for item in genres_list]
+    actors_list = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    [Actor.objects.create(first_name=actor[0], last_name=actor[1])
+     for actor in actors_list]
```

Comment:
Rewrite this list comprehension to not use indexes, it would be better to perform unpacking during the loop, e.g.
```suggestion
    [Actor.objects.create(first_name=first_name, last_name=last_name)
     for first_name, last_name in actors_list]
```